index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="style.css">
+		<title> My Portfolio </title>
+	</head>
+
+	<body>
+
+		<header id="header">
+			<nav id="navigation">
+				<ul>
+					<li><a href="index.html" id="logo"> My Logo</a></li>
+					<li><a href="" class="menu-link"><img src="iconMenu.svg" alt="menu-icon" class="menu-icon"></a></li>
+				</ul>
+			</nav>
+		</header>
+		<main class="presentation">
+			<div class="greeting">
+				<h2>I’m Mariane Glad to see you!</h2>
+			</div>
```

Comment:
- [ ] Excellent job implementing the changes suggested by reviewer 🥇👍 As suggested by the reviewer [here](https://github.com/fredo509/Portfolio_setup_mobile_first/pull/3#discussion_r1108903710) please make sure to use `h1` tag here in place of `h2`. `h1` tag is used for the most important heading/content of the page. It helps search engines to pick out topics from your page ⚒ Please fix this one 😃. To learn more, please visit [this link](https://www.reliablesoft.net/h1-tag/)